sdk/loadtesting/azure-developer-loadtesting/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.0.20 (Unreleased)
 
 ### Features Added
```

Comment:
When releasing, you'll want to remove as `###` sections with no entries. You'll also need to change `(Unreleased)` to `(YYYY-MM-DD)` with the date you'll be releasing.

Reply:
Got you, we will settle in all the changes required for all the SDKs.
Will create the release request separately, when we plan to release the SDKs (probably after the deployment freeze)